test/configs/FileResourceStoreConfig.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  AcceptPreferenceParser,
+  AclManager,
+  AuthenticatedLdpHandler,
+  CompositeAsyncHandler,
+  HttpHandler,
+  InteractionController,
+  MetadataController,
+  Operation,
+  QuadToTurtleConverter,
+  RepresentationConvertingStore,
+  ResourceStore,
+  ResponseDescription,
+  RuntimeConfig,
+  TurtleToQuadConverter,
+} from '../..';
+import { UnsecureWebIdExtractor } from '../../src/authentication/UnsecureWebIdExtractor';
+import { AllowEverythingAuthorizer } from '../../src/authorization/AllowEverythingAuthorizer';
+import { UrlBasedAclManager } from '../../src/authorization/UrlBasedAclManager';
+import { BasicRequestParser } from '../../src/ldp/http/BasicRequestParser';
+import { BasicResponseWriter } from '../../src/ldp/http/BasicResponseWriter';
+import { BasicTargetExtractor } from '../../src/ldp/http/BasicTargetExtractor';
+import { RawBodyParser } from '../../src/ldp/http/RawBodyParser';
+import { DeleteOperationHandler } from '../../src/ldp/operations/DeleteOperationHandler';
+import { GetOperationHandler } from '../../src/ldp/operations/GetOperationHandler';
+import { PostOperationHandler } from '../../src/ldp/operations/PostOperationHandler';
+import { PutOperationHandler } from '../../src/ldp/operations/PutOperationHandler';
+import { MethodPermissionsExtractor } from '../../src/ldp/permissions/MethodPermissionsExtractor';
+import { FileResourceStore } from '../../src/storage/FileResourceStore';
+import { ServerConfig } from '../configs/ServerConfig';
+
+// This is the configuration from bin/server.ts
+
+export class FileResourceStoreConfig implements ServerConfig {
```

Comment:
something like `x.getFileStore.withConverter(converter).withLocking(locker)`  or just more basic things? We have a possible idea that could build further on this but maybe we can discuss that tomorrow.

Reply:
I was just thinking of some sort of config util that had functions for the most common blocks, doesn't necessarily have to be something fancy that can be chained. The main goal is to make sure that if a new config needs to be written it can be done easily and without having to copy/paste to many large parts from other configs (because if that would be necessary why can't we have a generic call that does that thing). No strong opinion on how to exactly do it though.